scripts/index.js

```diff
@@ -0,0 +1,154 @@
+}
+
+const changeTempColor = () => {
+    const tempValue = document.querySelector("#tempValue");
+    const tempDisplay = document.querySelector("#tempDisplay");
+
+    if(state.temperature >= 80){
+        tempDisplay.style.color = weather_info.hot.fontColor
+        tempValue.style.color = weather_info.hot.fontColor
+    }else if(state.temperature <= 79 && state.temperature >=70){
+        tempDisplay.style.color = weather_info.warm.fontColor
+        tempValue.style.color = weather_info.warm.fontColor
+    }else if(state.temperature <= 69 && state.temperature >=60){
+        tempDisplay.style.color = weather_info.cool.fontColor
+        tempValue.style.color = weather_info.cool.fontColor
+    }else if(state.temperature < 59){
+        tempDisplay.style.color = weather_info.freezing.fontColor
+        tempValue.style.color = weather_info.freezing.fontColor
+    }
+}
+
+const changeLandscape = () => {
+    const lanscapeContent = document.querySelector("#lanscapeContent");
+    const landscapeContainer = document.querySelector("#landscapeContainer");
+    if(state.temperature >= 80){
+        landscapeContainer.style.color = weather_info.hot.color
+        lanscapeContent.textContent = `${weather_info.hot.emojis}`;
+
+    }else if(state.temperature <= 79 && state.temperature >=70){
+        landscapeContainer.style.color = weather_info.warm.color
+        lanscapeContent.textContent = `${weather_info.warm.emojis}`;
+
+    }else if(state.temperature <= 69 && state.temperature >=60){
+        landscapeContainer.style.color = weather_info.cool.color
+        lanscapeContent.textContent = `${weather_info.cool.emojis}`;
+
+    }else if(state.temperature < 59){
+        landscapeContainer.style.color = weather_info.freezing.color
+        lanscapeContent.textContent = `${weather_info.freezing.emojis}`;
+    }
+}
+
+const changeTempValue = () => {
+    const tempValue = document.querySelector("#tempValue");
+    tempValue.textContent = `Temperature: ${state.temperature}F°`;
+    const tempDisplay = document.querySelector("#tempDisplay");
+    tempDisplay.textContent = `${state.temperature}F°`;
+}
+
+const decreaseTemp = (event) => {
+    state.temperature -= 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const increaseTemp = (event) => {
+    state.temperature += 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const changeSky = (event) => {
+    const skyContent = document.querySelector("#skyContent");
+    const mainBackground = document.querySelector("#mainBackground");
+    const mainContent = document.querySelector(".mainContent");
+
+    if(event.target.options.selectedIndex == 0){
+        mainContent.style.backgroundColor = "#c9c9c3"
+        mainBackground.style.backgroundImage = `url(${sky_info.sunny})`
+        skyContent.textContent = `☁️ ☁️ ☁️ ☀️ ☁️ ☁️`;
```

Comment:
I'd love to see a more data-centric approach as you did for the temperature. If these values were stored in a data structure, we could both use iteration to find the object that has the values we want to apply to the UI, but we could also build the `option`s in the sky `select` from the same data!